### PR TITLE
RHDEVDOCS-5589: Pipelines 1.11.1 release notes

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

enterprise 4.12 and later

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-5589

Link to docs preview:

https://63241--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes#release-notes-1-11-1_op-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-release-notes-1-11.adoc
Comment (savitaashture) [inline modules/op-release-notes-1-11.adoc:235]: If i remember correctly i think we are not shipping cli in 1.11.1 so if thats the case we need to remove these release notes
cc @concaf, shverma@redhat.com

File: modules/op-release-notes-1-11.adoc
Comment (savitaashture) [inline modules/op-release-notes-1-11.adoc:229]: > After this update, {pac} CI/CD actions are executed correctly in this case.

How about adding what is shows after the update 
I mean before update it use to show `skipped` but after update it shows `Pending approval`

File: modules/op-release-notes-1-11.adoc
Comment (savitaashture) [inline modules/op-release-notes-1-11.adoc:235]: If i remember correctly i think we are not shipping cli in 1.11.1 so if thats the case we need to remove these release notes
cc @concaf, [shverma@redhat.com](mailto:shverma@redhat.com)

File: modules/op-release-notes-1-11.adoc
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:227]:
```suggestion
* Before this update, the `tekton-pipelines-controller` component used a config map named `config-leader-election`. This name is the default value for knative controllers, so the configuration process for {pipelines-shortname} could affect other controllers and vice versa. With this update, the component uses a unique config name, so the configuration process for {pipelines-shortname} does not affect other controllers and is not affected by other controllers.
```

File: modules/op-release-notes-1-11.adoc
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:229]:
```suggestion
* Before this update, when a user without write access to a GitHub repository opened a pull request, {pac} CI/CD actions would show as `skipped` in GitHub. With this update, {pac} CI/CD actions are shown as `Pending approval` in GitHub.
```

File: modules/op-release-notes-1-11.adoc
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:231]:
```suggestion
* Before this update, {pac} ran CI/CD actions for every pull request into a branch that matched a configured branch name. With this update, {pac} runs CI/CD actions only when the source branch of the pull request matches the exact configured branch name.
```

File: modules/op-release-notes-1-11.adoc
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:233]:
```suggestion
* Before this update, a `tkn pac resolve -f` command with a directory specified did not request a GitHub API token even when `{{ git_auth_secret }}` was present in the directory. With this update, the `tkn pac resolve -f` command requests a GitHub API token in this case.
```

File: modules/op-release-notes-1-11.adoc
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:235]:
```suggestion
* Before this update, the `tkn pac resolve -f` command failed if a cluster was running an older version of {pipelines-shortname}, which did not yet support the `v1` API. With this update, you can specify the `--v1beta` or `-B` flag in this command to use the `v1beta` API.
```

File: modules/op-release-notes-1-11.adoc
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:237]:
```suggestion
* Before this update, the `tkn pr logs` command failed to show logs of a pipeline run that uses the cluster resolver, hub resolver, bundle resolver, or git resolver. With this update, the `tkn pr logs` command shows logs of pipeline runs that use any type of resolver.
```

File: modules/op-release-notes-1-11.adoc
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:239]:
```suggestion
* Before this update, in {pipelines-title} 1.11.0, the Operator always installed {tekton-chains} and you would not disable installation of the {tekton-chains} component. With this update, you can set the value of the `disabled` parameter to `true` in the `TektonConfig` CR  to disable installation of {tekton-chains}.
```

File: modules/op-release-notes-1-11.adoc
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:241]:
```suggestion
* Before this update, if you configured {tekton-chains} on an older version of {pipelines-shortname} using the `TektonChain` CR and then upgraded to {pipelines-shortname} version 1.11.0, the configuration information was overwritten. With this update, if you upgrade from an older version of {pipelines-shortname}, {tekton-chains} configuration information is preserved.
```

**Review by shipsing — CHANGES_REQUESTED** [review]

File: modules/op-release-notes-1-11.adoc
Comment (mramendi) [reply modules/op-release-notes-1-11.adoc]: Fixed, thanks @shipsing

File: modules/op-release-notes-1-11.adoc
Comment (savitaashture) [inline modules/op-release-notes-1-11.adoc:233]: @mramendi this one also related to tkn-pac cli 
can we remove it

**Review by ppitonak — APPROVED** [review]

**Review by khrm — COMMENTED** [review]
/lgtm for Pipelines

**Review by concaf — APPROVED** [review]

**Comment by Preeticp** [comment]
Please note: This PR is ready to be reviewed for merging, but we are waiting for the go-live from the team to merge this hence the merge review needed and do not merge labels applied to this PR.

File: modules/op-release-notes-1-11.adoc
Comment (piyush-garg) [inline modules/op-release-notes-1-11.adoc:219]:
```
when a v1beta1 pipelinerun has pipelineref of bundle, while conversion to v1, the `kind` value was getting wrongly set to `Task`. The issue did not exist in previous versions of {pipelines-title}. With this update, the `kind` value is set correctly.
```

Comment (mramendi) [reply modules/op-release-notes-1-11.adoc:219]: @piyush-garg I don't understand this. In what use cases does a conversion from v1beta1 to v1 happen?

**Comment by bscott-rh** [comment]
> Please note: This PR is ready to be reviewed for merging, but we are waiting for the go-live from the team to merge this hence the merge review needed and do not merge labels applied to this PR.

Please squash down to 1 commit before you are ready to merge. Which versions of the openshift-docs repo should this be cherrypicked to? `main` only, or does it need to be picked to any of the enterprise-4.10+ branches? Thank you

File: modules/op-release-notes-1-11.adoc
Comment (PuneetPunamiya) [inline modules/op-release-notes-1-11.adoc:237]: We need to add one note here 

NOTE:- Data will be preserved only if in previous versions Tekton Chains was installed in the targetNamespace i.e. where TektonConfig was installed (e.g. `openshift-pipelines`)

File: modules/op-release-notes-1-11.adoc
Comment (piyush-garg) [inline modules/op-release-notes-1-11.adoc:219]: the `kind` value is getting set everytime, and during creating resources on cluster after conversion to v1

**Comment by PuneetPunamiya** [comment]
LGTM

**Comment by piyush-garg** [comment]
lgtm

**Comment by savitaashture** [comment]
LGTM

**Comment by mramendi** [comment]
This was peer-reviewed by @shipsing but there were changes after that and the point release is out, so we need peer review ASAP (and then I will squash and request merge)

File: modules/op-release-notes-1-11.adoc
Comment (tmalove) [inline modules/op-release-notes-1-11.adoc:205]:
```suggestion
* This update fixes the `tkn pr delete` command for correctly using the `keep` flag. With this update, if the value of the `keep` flag equals the number of associated task runs or pipeline runs, the `tkn pr delete` command returns exit code `0` along with a message.
```

File: modules/op-release-notes-1-11.adoc
Comment (tmalove) [inline modules/op-release-notes-1-11.adoc:219]:
```suggestion
* Before this update, in {pipelines-title} 1.11.0, when a pipeline run is defined using {pac}, the definition in the Git repository references the `tekton.dev/v1beta1` API version and includes a `spec.pipelineRef.bundle` entry, the `kind` parameter for the bundle reference was wrongly set to `Task`. The issue did not exist in earlier versions of {pipelines-title}. With this update, the `kind` parameter is set correctly.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#e

File: modules/op-release-notes-1-11.adoc
Comment (tmalove) [inline modules/op-release-notes-1-11.adoc:225]: Suggestios - Perhaps change 'latest' to 'current'. See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l

**Review by tmalove — APPROVED** [review]

**Comment by mramendi** [comment]
This PR is fully ready for merge review, merging, and cherry-picking. it is time-sensitive because it is release notes for a product update that already got released. Yesterday a merge review was done, but the release itself was pushed out to today and the RNs got a couple of changes since then. Peer review on the final version is done and everything is squashed.

**Comment by mramendi** [comment]
I seem to be unable to remove the do-not-merge label, help needed?

**Comment by bscott-rh** [comment]
Merged to main and cherrypicked to 4.12-4.14.

**Pull request merged**
